katas/content/superposition/zero_and_bitstring/Verification.qs

```diff
@@ -0,0 +1,31 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Katas;
+
+    operation ZeroAndBitstringSuperposition_Reference (qs : Qubit[], bits : Bool[]) : Unit is Adj + Ctl {
+        H(qs[0]);
+
+        for i in 1 .. Length(qs) - 1 {
+            if bits[i] {
+                CNOT(qs[0], qs[i]);
+            }
+        }
+    }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        let qubits = 3;
+        for i in 0 .. (2 ^ qubits) - 1 {
+            let bits = IntAsBoolArray(i, qubits);
```

Comment:
This approach generates some tests that don't fit the task description - "You are guaranteed that the first bit of the bit string is true.", but the very first test is [false, false, false]. We'll also need to check different array lengths, to make sure the loop conditions are specified correctly.

I would recommend having a list of ~6 hardcoded arrays, starting with [[true], [true, false], [true, true], ...], and a loop iterating through them and running the tests on just them. (This will also reduce the length of the log the learner gets if the solution is correct, since we report each success)

Reply:
good point! will adjust.